syntax = "proto3";

package test;

option cc_generic_services = true;

message ResultCode 
{
    int32 errcode = 1;
    bytes errmsg = 2;
}

message LoginRequest
{
    bytes username = 1;
    bytes password = 2;
}

message LoginResponse
{
    ResultCode result = 1;
    bool success = 2;
}

service UserServiceRPC 
{
    rpc Login (LoginRequest) returns (LoginResponse);
}